microprofile/metrics/src/main/java/io/helidon/microprofile/metrics/MetricsCdiExtension.java

```diff
@@ -137,6 +138,7 @@
     private final Set<Class<?>> metricsAnnotatedClasses = new HashSet<>();
     private final Set<Class<?>> metricsAnnotatedClassesProcessed = new HashSet<>();
     private final Map<Class<?>, Set<Method>> methodsWithSyntheticSimplyTimer = new HashMap<>();
+    private final Set<Method> syntheticSimplyTimersToRegister = new HashSet<>();
```

Comment:
Typo: should be `syntheticSimpleTimersToRegister` (not "simply"). `@SimplyTimed` is the annotation. (I know, a bit annoying.)